src/controller/Controller.js

```diff
@@ -0,0 +1,54 @@
+import InputView from '../view/InputView.js';
+import Car from '../model/Car.js';
+import OutputView from '../view/OutputView.js';
+
+export default class Controller {
+  constructor() {
+    this.carList = [];
+    this.MAX_MOVE = 0;
+  }
+
+  /**
+   * 자동차 이름 목록을 통해 자동차 객체를 생성합니다.
+   * @param {string[]} [자동차 이름 목록]
+   */
+  createCar(carNameList) {
```

Comment:
만약 이 코드의 의도가 자동차 한 대가 아닌 여러 자동차를 만들어내는 거라면 복수형으로 표현해주는 건 어떨까요?
```suggestion
  createCars(carNameList) {
```

Reply:
이 부분도 Parkhanyoung님의 의견이 조금 더 명확한 표현인 것 가네요! 의견 감사합니다 :)